Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -89,6 +89,11 @@ For more details on the mons and when to choose a number other than `3`, see the
 * `disruptionManagement`: The section for configuring management of daemon disruptions
     * `managePodBudgets`: if `true`, the operator will create and manage PodDisruptionBudgets for OSD, Mon, RGW, and MDS daemons. OSD PDBs are managed dynamically via the strategy outlined in the [design](https://github.com/rook/rook/blob/master/design/ceph/ceph-managed-disruptionbudgets.md). The operator will block eviction of OSDs by default and unblock them safely when drains are detected.
     * `osdMaintenanceTimeout`: is a duration in minutes that determines how long an entire failureDomain like `region/zone/host` will be held in `noout` (in addition to the default DOWN/OUT interval) when it is draining. This is only relevant when  `managePodBudgets` is `true`. The default value is `30` minutes.
+    * `pgHealthCheckTimeout`: A duration in minutes that the operator will wait for the placement groups to become healthy (see `pgHealthyRegex`) after a drain was completed and OSDs came back up.
+    Operator will continue with the next drain if the timeout exceeds. It only works if `managePodBudgets` is `true`.
```

Comment:
All settings in this section would only apply if `managePodBudgets: true`, so I think no need to mention it here.
```suggestion
    Operator will continue with the next drain if the timeout exceeds.
```